### DISCUSSION TITLE
[Project Beta] Keep issues' order across board columns

### DISCUSSION THREAD

**Discussion by Dyljyn** [community/community#36763] [Projects and Issues]
**Use case**
While moving issues ordered by their priority[^1] from column to column to update their status[^1], the order of priority easily gets lost because

1. issues can be placed in any vertical position within a column;
2. the order related to the issues in the new column is unavailable and most probably forgotten by the user.

[^1]: a field within the project

**Comment by ossanna16** [comment]
Thank you for asking a good question @Dyljyn, I'm glad you found our community 🙂
